### PR TITLE
Add admin support for user primary key as uuid

### PR DESCRIPTION
**Pull request by lukewiwa**
I noticed this route would error out since the regex only looks for digits. Here's a fix.

### CODE REVIEW

File: guardian/admin.py [hunk 1]
```diff
@@ -102,11 +102,11 @@ def get_urls(self):
                     view=self.admin_site.admin_view(
                         self.obj_perms_manage_view),
                     name='%s_%s_permissions' % info),
-                url(r'^(?P<object_pk>.+)/permissions/user-manage/(?P<user_id>\-?\d+)/$',
+                url(r'^(?P<object_pk>.+)/permissions/user-manage/(?P<user_id>.+)/$',
                     view=self.admin_site.admin_view(
                         self.obj_perms_manage_user_view),
                     name='%s_%s_permissions_manage_user' % info),
-                url(r'^(?P<object_pk>.+)/permissions/group-manage/(?P<group_id>\-?\d+)/$',
+                url(r'^(?P<object_pk>.+)/permissions/group-manage/(?P<group_id>.+)/$',
```
Comment (jpic) [inline guardian/admin.py]: why not make the move to path while were at it ?

**Comment by ad-m** [comment]
PR is not possible to merge because it contains everything. @lukewiwa , can you rebase? Users in issue #611 awaiting for that PR

**Comment by lukewiwa** [comment]
Looks like this has been resolved in the latest devel branch by changing the urls to paths anyway. Should I still rebase then? Thinking it might not be worth it.

**Comment by ad-m** [comment]
@lukewiwa , 10 hour ago I implemented that by self (see #634). 6 hour ago I released new version which includes that changes. Any feedback welcome!

**Comment by ad-m** [comment]
BTW. Do you have still any needs to keep own fork?

**Comment by lukewiwa** [comment]
@ad-m nah will delete

**Pull request closed without merging**